### DISCUSSION TITLE
Font colors of headers in exported PDFs or HTML

### DISCUSSION THREAD

**Discussion by rpace314159** [Zettlr/Zettlr#2749] [Q&A] [answered]
When I export to a PDF or HTML, the color of the H1 header is some shade of orange (HTML) or tan/brown (PDF).  How do I control the font color?  e.g. I would just like to have a black font color.

**Comment by nathanlesage** [comment, marked as answer]
First, if you need to export to PDF often, it is recommended to install LaTeX and use the exporter PDF (LaTeX). The PDF Document exporter is simply an HTML exporter with following PDF exporting. The default template could definitely use a make over. In order to change it, you'd need to provide your own template in the HTML defaults file.

**Reply by rpace314159** [reply]
Thanks.  I do have LaTeX installed.  The PDF(LaTeX) exporter produces a black font color for all headers, as desired.  In hindsight, I suppose I may be discovered this as I intended to use some LaTeX in my document anyway, but had not progressed that far. :smiley: :+1:

**Reply by sweeney97** [reply]
I thought that too: the colors are kind of odd

I was able to change links color for the PDF export via Yaml but it didn't work for the HTML export.

According to the [documentation](https://docs.zettlr.com/en/core/export/), there is two way of customizing exports. Is it complicated to make a HTML template only to change links color? Do you know if there is more documentation abouti somewhere?

Thanks a lot!

**Reply by nathanlesage** [reply]
Quick notice: Yesterday I finally dumped the custom HTML template, so now Zettlr uses Pandoc's default template. See if that fixes it for you (comes in the next patch). If not, adapting an HTML template should be fairly straightforward, provided you have a basic knowledge of HTML and CSS

**Reply by rpace314159** [reply]
I'm happy to try the patch when it is available.  Not sure where to get the updated code, though, when available. "Zettlr Nightlies" page shows last update 1 Nov 2021.  I use the x86_64 AppImage file on SuSE Tumbleweed.

**Reply by nathanlesage** [reply]
Nightlies get compiled every Monday, so tomorrow is the next one. I think during the week there'll also be 2.0.3

**Reply by sweeney97** [reply]
I like Pandoc's default template, happy to know that it'll be used in the next patch!

Thank you @nathanlesage for this quick answer :)
